ui/tests/unit/services/permissions-test.js

```diff
@@ -0,0 +1,137 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import Pretender from 'pretender';
+
+const PERMISSIONS_RESPONSE = {
+  data: {
+    exact_paths: {
+      foo: {
+        capabilities: ['read'],
+      },
+      'bar/bee': {
+        capabilities: ['create'],
+      },
+      boo: {
+        capabilities: ['deny'],
+      },
+    },
+    glob_paths: {
+      'baz/biz': {
+        capabilities: ['read'],
+      },
+    },
+  },
+};
+
+module('Unit | Service | permissions', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.server = new Pretender();
+    this.server.get('/v1/sys/internal/ui/resultant-acl', () => {
+      return [200, { 'Content-Type': 'application/json' }, JSON.stringify(PERMISSIONS_RESPONSE)];
+    });
+  });
```

Comment:
Just a different approach here. You could make all these tests more isolated by injecting a mocked `store` into your service, something like:

```javascript
adapterFor() {
  return {
    query() {
      return Promise.resolve(PERMISSIONS_RESPONSE);
    }
  }
}
```

Then you wouldn't need to use Pretender, and therefore your tests are more isolated/testing the unit, i.e. they only run the code you have written and are testing.